plugins/dfsound/externals.h

```diff
@@ -115,6 +115,8 @@ typedef struct
  int               iRightVolume;                       // right volume
  ADSRInfoEx        ADSRX;
  int               iRawPitch;                          // raw pitch (0...3fff)
+ unsigned char	   IgnoreLoop;

```

Comment:
Can you move this up a bit and make it a bitfield? Like
```unsigned int    bIgnoreLoop:1```
It will then be stored in the same int as `prevflags` etc and take no extra space.